src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -519,6 +520,7 @@ public synchronized void updateEntity(DbEntity entity) throws DatabaseEngineExce
                 if (!toRemove.isEmpty()) {
                     if (properties.allowColumnDrop()) {
                         dropColumn(entity, toRemove.toArray(new String[toRemove.size()]));
+                        // TODO: drop indexes
```

Comment:
Not forgotten. PDB doesn't have yet the functionality to drop indexes, it does not do it anywhere, but should do it here. It's more of a reminder.